nevergrad/functions/games/game.py

```diff
@@ -404,13 +402,12 @@ def _simulate_game(self, x: np.ndarray) -> float:
         # FIXME: an adaptive opponent, e.g. bandit, would be better.
         # We play a game as player 1.
         np_state = np.random.get_state()
-        np.random.seed(self.instrumentation.random_state.randint(12560, dtype=np.uint32))
         p1 = x[:(self.dimension // 2)]
-        p2 = self.instrumentation.random_state.normal(size=self.dimension // 2)
+        p2 = np.random.normal(size=self.dimension // 2)
```

Comment:
Using instrumentation indeed does not work, I'll try to update this when I simplify the `InstrumentedFunction` mess to something way simpler